### DISCUSSION TITLE
Hot words

### DISCUSSION THREAD

**Discussion by sspaek** [coqui-ai/STT#2344] [General]
Hello everyone, I see the following in the documentation:
"It is worth noting that boosting non-existent words in scorer (mostly proper nouns) or a word that share no phonetic prefix with other word in the input audio don’t change the final transcription."

Could you please confirm that the above means that there is no point adding hot words that don't already exist in the "scorer", and will not be "boosted"?

Thanks very much

**Comment by HarikalarKutusu** [comment]
AFAIK, they must exist in the scorer. You can only modify the probabilities of existing words. So you need to upgrade your scorer with those words you want to be recognized.
